clients/admin-ui/src/features/config-wizard/OrganizationInfoForm.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Stack,
+  Text,
+  Tooltip,
+} from "@fidesui/react";
+import { useFormik } from "formik";
+import type { NextPage } from "next";
+import React, { useState } from "react";
+
+import { QuestionIcon } from "~/features/common/Icon";
+
+const useOrganizationInfoForm = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const formik = useFormik({
+    initialValues: {
+      organizationName: "",
+      organizationDescription: "",
+    },
+    onSubmit: async () => {
+      setIsLoading(true);
+      // submission API call when available
```

Comment:
Doesn't `POST /api/v1/organization` and `PUT /api/v1/organization` work for this use case? It seems like we should setup this form to automatically edit the default organization (@SteveDMurphy has experience here!) and wire up the API callls?

Reply:
API endpoints are all set up for org/system, so they should get wired up in these PRs as well

Reply:
@ThomasLaPiana @NevilleS - Correct me if I'm wrong, but we won't have the entire body to send for a valid request to the [API here](https://ethyca.github.io/fides/api/) when we create an organization until the end of the wizard?

Reply:
At this step, we only have the name and description of the org

Reply:
@LKCSmith technically the only thing that an organization needs to be valid is a `fides_key`, so the organization can actually be created at this step

Reply:
@ThomasLaPiana Which API call will allow me to access the `fides_key`?

Reply:
For this step, we've decided to overwrite the `default_organization` (as discussed with @NevilleS & @SteveDMurphy ), so it should be setting `fides_key` to `default_organization` when sending the API call